src/CopySignUpSheetToRoles.js

```diff
@@ -87,31 +93,41 @@ class SpeechDetails {
   }
 }
 
+/// Slice a list based on the location of two entries in the list inclusive.
+/// Throws an exception if the resulting slice is empty. Assumes all entires are
+/// unique.
+function between(arr, first, last) {
```

Comment:
nit: I slightly prefer more descriptive names over/in addition to more descriptive comments.

`getValuesBetween` or similar would let most people skip reading the comments and would let reading the rest flow more smoothly